packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -873,76 +559,32 @@ class FlutterDriver {
   ///     ]
   ///
   /// [getFlagList]: https://github.com/dart-lang/sdk/blob/master/runtime/vm/service/service.md#getflaglist
-  Future<List<Map<String, dynamic>>> getVmFlags() async {
-    await _restorePeerConnectionIfNeeded();
-    final Map<String, dynamic> result = await _peer.sendRequest('getFlagList') as Map<String, dynamic>;
-    return result != null
-        ? (result['flags'] as List<dynamic>).cast<Map<String,dynamic>>()
-        : const <Map<String, dynamic>>[];
-  }
+  ///
+  /// Calling this function on WebFlutterDriver will throw unimplemented error.
+  Future<List<Map<String, dynamic>>> getVmFlags();
 
   /// Starts recording performance traces.
   ///
   /// The `timeout` argument causes a warning to be displayed to the user if the
   /// operation exceeds the specified timeout; it does not actually cancel the
   /// operation.
+  ///
+  /// For WebFlutterDriver, this is only supported for Chrome.
   Future<void> startTracing({
-    List<TimelineStream> streams = _defaultStreams,
+    List<TimelineStream> streams = const <TimelineStream>[TimelineStream.all],
     Duration timeout = kUnusuallyLongTimeout,
-  }) async {
-    assert(streams != null && streams.isNotEmpty);
-    assert(timeout != null);
-    try {
-      await _warnIfSlow<void>(
-        future: _peer.sendRequest(_setVMTimelineFlagsMethodName, <String, String>{
-          'recordedStreams': _timelineStreamsToString(streams),
-        }),
-        timeout: timeout,
-        message: 'VM is taking an unusually long time to respond to being told to start tracing...',
-      );
-    } catch (error, stackTrace) {
-      throw DriverError(
-        'Failed to start tracing due to remote error',
-        error,
-        stackTrace,
-      );
-    }
-  }
+  });
 
   /// Stops recording performance traces and downloads the timeline.
   ///
   /// The `timeout` argument causes a warning to be displayed to the user if the
   /// operation exceeds the specified timeout; it does not actually cancel the
   /// operation.
+  ///
+  /// For WebFlutterDriver, this is only supported for Chrome.
```

Comment:
Here and elsewhere, use correct dartdoc formatting.

```suggestion
  /// For [WebFlutterDriver], this is only supported for Chrome.
```